src/main/scala/scala/compat/java8/collectionImpl/Stepper.scala

```diff
@@ -0,0 +1,545 @@
+package scala.compat.java8.collectionImpl
+
+import java.util.Spliterator
+
+/** A Stepper is a specialized collection that can step through its
```

Comment:
How does that differ from using the existing `.iterator` method to get an Iterator for a collection?